setup.py

```diff
@@ -16,7 +16,7 @@
     'icalendar>=4.0.3',
     'urwid>=1.3.0',
     'pyxdg',
-    'pytz',
+    "backports.zoneinfo;python_version<'3.9'",
```

Comment:
Please add `types-backports` to mypy's [additional dependencies](https://github.com/pimutils/khal/blob/16e9540f3bc51cebfae3015898cbee645d4bfd92/.pre-commit-config.yaml#L39) so that mypy checks pass with the new dependency.